src/Qir/Execution/Tests.Microsoft.Quantum.Qir.Tools/TestCases/FullStateDriverGenerator/UseBoolArg.cpp

```diff
@@ -0,0 +1,87 @@
+//------------------------------------------------------------------------------
+// This code was generated by a tool.
```

Comment:
A note from future developers of this code: finding this comment in a file that you want to modify is very frustrating. If possible it would be nice to say which tool generates it and how.

Reply:
I added information about the tool that generated the file and its purpose.